spec/models/timeline_entry_spec.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+RSpec.describe TimelineEntry do
```

Comment:
Do we need this if it's covered by a feature spec?

Reply:
It's nice to have this checked, there might be things that haven't been feature-tested (yet).